compiler/AST/expr.cpp

```diff
@@ -3719,6 +3719,7 @@ CallExpr::insertAtTail(BaseAST* ast) {
     argList.insertAtTail(new SymExpr(a));
   else
     argList.insertAtTail(toExpr(ast));
+  // TODO -- should this call parent_insert_help ?
```

Comment:
This addition of a comment seems out of the blue.
I would prefer posing that question outside this PR.